admin/app.js

```diff
@@ -19,9 +19,23 @@ const session = require('express-session')
 const MongoStore = require('connect-mongo')(session)
 const breadcrumbs = require('express-breadcrumbs')
 const fs = require('fs')
+const config = require('./config')
+const whitelist = require('./whitelist')
+
+const unsetVars = []
+Object.keys(config).map((key) => {
+  if (!config[key] && !whitelist.includes(key)) {
```

Comment:
I don't think there should be any concept of a whitelist for production - any missing config value should terminate.  The whitelist vars should really be devWhitelist.